dm/worker/config.go

```diff
@@ -171,6 +172,11 @@ func (c *Config) adjust() error {
 		}
 	}
 
+	if c.Name == "" {
+		log.L().Info("worker name is not given, we will set AdvertiseAddr as the worker name", zap.String("AdvertiseAddr", c.AdvertiseAddr))
```

Comment:
_logger_ may not be initialized when reaching here. (_main.go_ -> Config.Parser -> Config.adjust)